src/payment/strategies/sezzle/sezzle-payment-strategy.ts

```diff
@@ -0,0 +1,63 @@
+import { FormPoster } from '@bigcommerce/form-poster';
```

Comment:
This seems like the same strategy as LayBuy. Wouldn´t it be better to refactor both strategies into a single strategy? @clopezh @josesonora